lib/external_interaction/sky_outgoing_calls.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:convert';
+import 'package:sky/native_interaction/rubix/rubix_util.dart';
+import 'package:http/http.dart' as http;
+
+
+Future<bool> fireAuthCallback({required String did,required String peerId,required String callBackUrl,required Token orgAccessToken,required String sessionId,required String chain })async {
+    var bodyJsonStr = jsonEncode({
+      'did': '$did.$peerId',
+      'session_id': sessionId,
+      'token': orgAccessToken,
```

Comment:
orgAccessToken is of Token class, we just need the token value inside it which is a string, that is what we can pass through an api.